README.md

```diff
@@ -96,6 +96,17 @@ Before running, adjust the following flags:
 * `--input` specify the full path to the input.tsv
 * `--output` name a directory to output the fastqc files to
```

Comment:
Add to the README that the `--output` directory will me created by nextflow if it does not exist